Classes&Inheritance.js

```diff
@@ -0,0 +1,112 @@
+"use strict";
+
+//Parent class
+function Builder(startValue) {
+    this.value = startValue;
+}
+
+//Common methods for str and int
+Builder.prototype.get = function () {
+    return this.value;
+}
+
+Builder.prototype.plus = function () {
+    for (var i = 0; i < arguments.length; i++) {
+        this.value += arguments[i];
+    }
+    return this;
+}
+
+Builder.prototype.minus = function () { }
+Builder.prototype.multiply = function () { }
+Builder.prototype.divide = function () { }
+
+// Child Class - StrBuilder in ES5
+function StringBuilder(startValue) {
+    Builder.call(this, startValue || "");
```

Comment:
```suggestion
    Builder.call(this, startValue || '');
```
It is not very important, but usually in JS we use '' or ``.